packages/vant/src/search/Search.tsx

```diff
@@ -119,13 +119,16 @@ export default defineComponent({
       });
 
       const onInput = (value: string) => emit('update:modelValue', value);
+      const hasErrorMessageClass =
+        fieldAttrs.errorMessage && 'has-error-message';
```

Comment:
Vant use BEM, please rename `has-error-message` to `van-search__field--with-message` and use the `bem` utility.

Reply:
ok